tests/Integration/DataManagementSystem/Test_StorageElement.py

```diff
@@ -0,0 +1,135 @@
+"""This module contains tests for the (DIP) StorageElement
+
+The tests upload gets a directory
+
+Requirements:
+
+* Running StorageElement instance, containing an LFN folder '/Jenkins', writable by the jenkins_user proxy
+* A jenkins_user proxy
+
+"""
+
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import os
+import shutil
+import sys
+import tempfile
+import time
+
+import pytest
+
+from pprint import pprint, pformat
+
+from DIRAC.Core.Base.Script import parseCommandLine
+
+parseCommandLine()
+
+from DIRAC.Core.Security.ProxyInfo import getProxyInfo  # noqa: E402
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getVOForGroup  # noqa: E402
+
+
+@pytest.fixture(scope="module")
+def se():
+    from DIRAC.Resources.Storage.StorageElement import StorageElementItem as StorageElement
+
+    return StorageElement("SE-1")
+
+
+try:
+    res = getProxyInfo()
+    if not res["OK"]:
+        raise Exception(res["Message"])
+
+    proxyInfo = res["Value"]
+    username = proxyInfo["username"]
+    vo = getVOForGroup(proxyInfo["group"])
+
+    if not vo:
+        raise ValueError("Proxy has no VO")
+
+    LFN_PATH = "/%s/test/unit-test/StorageElement/" % vo
+
+except Exception as e:  # pylint: disable=broad-except
+    print(repr(e))
+    sys.exit(2)
+
+
+@pytest.fixture
+def tempFile():
+    """Create unique temprary file, will be cleaned up at the end."""
+    theTemp = tempfile.NamedTemporaryFile("w")
+    theTemp.write(str(time.time()))
+    theTemp.seek(0)  # write the characters to file
+    yield theTemp.name
+    theTemp.close()
+
+
+@pytest.fixture
+def localDir():
```

Comment:
I think this can be replaced with pytest's [`tmp_path`](https://docs.pytest.org/en/6.2.x/reference.html#std-fixture-tmp_path) fixture

Reply:
Nice!